examples/supertuxkart/README.md

```diff
@@ -0,0 +1,11 @@
+# Super Tux Kart Example
+
+Example using a [Super Tux Kart](https://supertuxkart.net/) dedicated game server.
+
+This example wraps the Super Tux Kart server with a [Go](https://golang.org) binary, and introspects
+stdout and a log file to provide the event hooks for the SDK integration.
```

Comment:
nit: does it actually watch stdout? or just the log file?

Reply:
Since removing the interceptor, we are now just looking at the log file. Comment updated!

Reply:
I don't see this update in the latest commit.

Reply:
Ah! I updated the main.go, but not the readme!